src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,12 +34,18 @@ internal TagHelperOutput(string tagName)
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
         public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
+            : this(tagName, attributes, HtmlEncoder.Default)
```

Comment:
@NTaylorMullen who is "they"?  tag helper authors aren't the people writing `Startup` classes for actual apps.  if we leave this, the authors will break their users.  and we could have prevented it.